prow-configurations/create-gcp-secrets.sh

```diff
@@ -8,7 +8,7 @@ usage () {
 }
 
 CURRENT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-FILES=("sa-gke-kyma-integration" "sa-vm-kyma-integration")
+FILES=("sa-gke-kyma-integration" "sa-vm-kyma-integration" "sa-gcs-plank" "sa-gcr-push")
```

Comment:
please document what roles these service accounts should have. Docu will be development/README.md soon. Have a look here: https://github.com/kyma-project/test-infra/pull/80

Reply:
Done